jkq-ddsim/README.md

```diff
@@ -0,0 +1,26 @@
+# JKQ-DDSIM Benchmark
+
+This benchmark uses Google Benchmark to measure the runtime of the jkq-ddsim simulator.
+
+For more information visit [the university site](http://iic.jku.at/eda/research/quantum_simulation) or the [GitHub repo](https://github.com/iic-jku/ddsim).
```

Comment:
could you add a link to the actual benchmark implementation here so people could find how the implementation was implemented if they are interested in comparing the difference between API design.

Reply:
Thanks! I added the link and removed the json/csv files.